src/components/Input.js

```diff
@@ -0,0 +1,32 @@
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import React from 'react';
+
```

Comment:
I was wondering how to implement a search feature component and if it would be beneficial for how the site functioned.  I was wondering does having this in a separate component help or is it just developers preference